test/Microsoft.TestPlatform.TestHostProvider.UnitTests/Hosting/DotnetTestHostManagerTests.cs

```diff
@@ -616,6 +616,9 @@ public void GetTestHostProcessStartInfoShouldIncludeTestHostPathFromSourceDirect
         StringAssert.Contains(startInfo.Arguments, expectedTestHostPath);
     }
 
+    // TODO: This assembly was previously compiled as net472 and so it was skipped and only ran as netcoreapp2.1. This fails in test, but works in code that is not isolated in appdomain. Might be worth fixing because we get one null here, and another in DotnetTestHostManager.
```

Comment:
I don't know if that's better but maybe we could conditionally put a `Ignore` attribute with the explanation? At least we would see ignored tests and that might drive us to fix it.

Reply:
Maybe, but no-one is reviewing ignored tests now, similarly to no-one looking at TODOs (at least not systematically). We should drive both to 0, so it does not really matter which way we will get to this.

Reply:
Keeping as is.